src/MissingTableHelper.php

```diff
@@ -70,9 +74,11 @@ private static function fillMissingTableMapping(
             $csvFile = new CsvFile($baseFileName . $name);
             $csvFile->writeRow($columns);
             $manifest = [
-                'destination' => 'in.c-' . $outputBucket . '.' . $name,
```

Comment:
Ked to ma `default bucket` tak neni potreba `destination` v manifeste?

Reply:
- jj, kdyz ma default bucket, tak se vysledny ID vygeneruje pro OM (vygeneruje se nazev bucketu a komponenta doda jen nazev tabulky) `pokus.csv`, takze do destination neni co zapsat (destination = tableid)
- kdyz nema default bucket, tak musi komponenta vygenerovat nazev ve tvaru `stage.bucket.table` a tim muze bud pojmenovat ten soubor `in.c-bucket.table.csv` nebo to zapsat do destination (generic dela oboji)